webapp-client/src/component/AddCategoryForm.js

```diff
@@ -6,7 +6,7 @@ export default class AddCategoryForm extends React.Component {
     e.preventDefault();
```

Comment:
Bump on removing the comment above

Reply:
Sorry, I think this is old content in my branch

Reply:
Actually I think this is addressed in another PR